packages/project-account-switcher/src/presenters/ProjectAccountSwitcherPresenter.js

```diff
@@ -135,7 +135,7 @@ export default class ProjectAccountSwitcherPresenter extends Component {
           <Typography>{label}</Typography>
         </div>
         <div className={css(styles.targetCaret)}>
-          <Caret24 />
+          <CaretDownMUI />
```

Comment:
Just replace `Caret24` with `CaretDownMUI`. 

The ProjectAccountSwitcher currently does not consume `ThemeContext`, thus the caret icon will not change from CaretDownMUI to SUI when density changes. 

It might be better to consume ThemeContext as other components, and change the icons when density changes, but it will then require to add `theme-data` and `theme-context` as dependencies. Hi @wmui51 what's your suggestions?

Reply:
For now lets not add any new dependencies for this, we can revisit this if this needs to be changed as we go.